sdk/eventhub/event-hubs/samples/browserSample.html

```diff
@@ -0,0 +1,37 @@
+<!-- 
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT Licence.
+  
+This sample demonstrates a simple send event operation.  
+Refer to other samples and replace code in main() to try them out -->
+
+<!-- Line that imports the SDK source script for use in browser -->
+<script src="../browser/event-hubs.js"></script>
+
+<script>
+  // EventHubClient holds reference to the EventHubClient global object exposed by the imported script
+  const EventHubClient = Azure.Messaging.EventHub.EventHubClient;
```

Comment:
Did you test this sample with the latest changes in this PR? Shouldnt `Azure.Messaging.EventHub` be `Azure.Messaging.EventHubs`

Reply:
Yes, I think I had edited it before pushing. Tested before pushing latest changes.